pkg/featureflags/client_capability.go

```diff
@@ -0,0 +1,52 @@
+package featureflags
```

Comment:
Wasn't sure if this was the most appropriate place for this logic or not -- open to suggestions